packages/terra-application-layout/package.json

```diff
@@ -42,7 +42,8 @@
     "terra-modal-manager": "^6.33.0",
     "terra-navigation-layout": "^5.23.0",
     "terra-navigation-side-menu": "^2.30.0",
-    "terra-popup": "^6.33.0"
+    "terra-popup": "^6.33.0",
+    "terra-theme-context": "^1.2.0"
```

Comment:
As you're adding this dependency, make sure that all packages are on *exactly* the same version i the lerna repo. There is a quirk with lerna hoisting that won't resolve the minor versions which will result in multiple versions of the context being pulled into the test site, breaking some of the components.

Reply:
Fixed 4e8bf43